installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3084,6 +3084,32 @@ delete from pac_config_properties where cfkey = 'shipper.attributes.to.preserve'
 
 update pac_v2_ui_options set optionURL="/admin/accounts/filter/attribute?attribute=assets" where filterId=12 and optionName ="Assets";
 
+INSERT IGNORE INTO pac_v2_ui_filters (filterId,filterName) VALUES (17,'admin-policy');
```

Comment:
can we use `INSERT ... ON DUPLICATE KEY UPDATE` instead?

Reply:
if the filter id is present already, we need to use a different filter id, otherwise filters for a different screen will be overriden.

Reply:
Insert..update would update the entry in case the filterId is already present and we dont want that. We cannot update the filter options for an already existing screen, it will mess up the screen. We have written a delete table for the filters option before executing any inserts, which will make sure any incorrect insert can be deleted in next run.